modules/waypoint_names_to_coordinates.py

```diff
@@ -0,0 +1,59 @@
+def waypoint_names_to_coordinates(waypoint_names: list[str],
+                                  waypoint_mapping: dict[str, tuple[float, float]]) \
+        -> tuple[bool, list[tuple[float, float]]]:
+    coordinates = []
+    for name in waypoint_names:
+        if name in waypoint_mapping:
+            coordinates.append(waypoint_mapping[name])
+        else:
+            return False, None
+    return True, coordinates
```

Comment:
If waypoint_names is an empty list, the function will return `True, []` . Does this seem right?